specific/account_asset_management_enhanced/models/account_asset_line.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from openerp import fields, models, api
+
+
+class AccountAssetLine(models.Model):
+    _inherit = 'account.asset.line'
+
+    @api.multi
+    def write(self, vals):
+        write_ctx = self._context
+        if 'allow_asset_line_update' in write_ctx:  # prevent maximum recursive
```

Comment:
Should we use,

ctx.get() to get the context value? Rather than just check whether the variable exsist...

Reply:
Overall, please try make the code more concise, if you can.